tests/dialects/test_clickhouse.py

```diff
@@ -12,6 +12,10 @@ class TestClickhouse(Validator):
     dialect = "clickhouse"
 
     def test_clickhouse(self):
+        self.validate_all(
+            "SELECT CONCAT('ABC', COALESCE(NAME, ''))", read={"oracle": "SELECT 'ABC' || NAME"}
+        )
```

Comment:
Always make sure to test roundtrip when adding tests. So your test should look like:

```python
self.validate_all(
    transpiled
    read={
        "oracle": input,
    },
    write={
        "clickhouse": roundtripped_transpiled,
        "oracle": transpiled_to_oracle,  # good to have (optional), tests for ClickHouse -> Oracle, if supported
    },
)
```

Reply:
Thanks. Another request– can you please move the test under `test_clickhouse`? Group it with the other `validate_all` checks for consistency (~L206).